packages/skygear-web/src/index.iife.ts

```diff
@@ -1,3 +1,14 @@
 import "core-js/features/promise";
 import "whatwg-fetch";
-export * from ".";
+
+import * as all from ".";
+// eslint-disable-next-line no-duplicate-imports
+import defaultContainer from ".";
+
+const merged = defaultContainer;
+for (const key of Object.keys(all)) {
+  // @ts-ignore
+  merged[key] = all[key];
```

Comment:
Is it intentional that developer can use `skygear.default` to access the default container? i.e. `skygear === skygear.default.default.default.default`

Reply:
It is a consequence of ES6 Module.

```typescript
export default a;
```

is equivalent to 

```typescript
export { a as default };
```

Reply:
I mean is this consequence intentional/desirable?

Reply:
I would not say it is intentional because the intention here is to merge the container and the namespace so that using default container should appear more nature. I dont think it is a problem that they are circular.